tmsc/__main__.py

```diff
@@ -33,6 +35,10 @@ def main():
     parser.add_argument("--prune-df", default=20, type=int,
                         help="Minimum number of times an identifer must occur in different "
                              "documents to be taken into account.")
+    parser.add_argument("--index_repo", default="https://github.com/src-d/models",
```

Comment:
These two are normally hardcoded in `modelforgecfg.py` which exists in sourced-ml. Users should be abstracted from these details.

I think that
```python
args.topics = Topics(log_level=args.log_level).load(source=args.topics)
```

will work - the backend will be created automatically. If it doesn't then there is a bug somewhere.

Reply:
✅ confirmed

```python
import logging
from sourced.ml.models import Topics
topics = Topics(log_level=logging.INFO).load(source=None)
```

works as expected! 🎉 thank you for notice